app/uk/gov/hmrc/eoricommoncomponent/frontend/playext/mappers/DateTuple.scala

```diff
@@ -16,58 +16,79 @@
 
 package uk.gov.hmrc.eoricommoncomponent.frontend.playext.mappers
 
+import java.time.Year
+
 import org.joda.time.LocalDate
 import play.api.data.Forms.{optional, text, tuple}
 import play.api.data.Mapping
 import uk.gov.hmrc.play.mappers.DateFields._
 
-import scala.util.Try
+import scala.util.{Success, Try}
 
 object DateTuple {
 
   def dateTuple(
-    validate: Boolean = true,
-    invalidDateError: String = "cds.error.invalid.date.format"
+    invalidDateError: String = "cds.error.invalid.date.format",
+    minYear: Int
   ): Mapping[Option[LocalDate]] = {
     def tuple2Date(tuple: (Option[String], Option[String], Option[String])) = tuple match {
-      case (Some(y), Some(m), Some(d)) =>
-        try Some(new LocalDate(y.trim.toInt, m.trim.toInt, d.trim.toInt))
+      case (Some(day), Some(month), Some(year)) =>
```

Comment:
You changed the order of values. Is it intentional?

Reply:
Yes.  It affects the order in which multiple errors are reported. E.g.
![image](https://user-images.githubusercontent.com/46934286/98549337-8d87f800-2292-11eb-8079-9e3aa93bf2be.png)

Reply:
okay, so it's working correctly